.github/workflows/build_wheels.yml

```diff
@@ -0,0 +1,141 @@
+name: Build python wheels
+
+on:
+  push:
+    branches:
+      - master
+      - release-*
+    tags:
+      - v*
+
+jobs:
+
+  build_source:
+    runs-on: ubuntu-18.04
```

Comment:
Should we use ubuntu-20.04 instead?

(Apparently this is now supported: https://github.com/actions/virtual-environments/issues/228)

Reply:
I found information in the documentation ([source](https://help.github.com/en/actions/reference/virtual-environments-for-github-hosted-runners#supported-runners-and-hardware-resources)) which says:

> Note: The Ubuntu 20.04 virtual environment is currently provided as a preview only. The ubuntu-latest YAML workflow label still uses the Ubuntu 18.04 virtual environment.

In this case using `ubuntu-latest` label may be a good idea. WDYT?